src/platform_impl/windows/window.rs

```diff
@@ -1008,22 +1008,24 @@ where
     // WindowFlags::VISIBLE and MAXIMIZED are set down below after the window has been configured.
     window_flags.set(WindowFlags::RESIZABLE, attributes.resizable);
 
-    let parent = match pl_attribs.parent {
-        Parent::ChildOf(parent) => {
+    let parent = match attributes.parent_window {
+        Some(RawWindowHandle::Win32(handle)) => {
             window_flags.set(WindowFlags::CHILD, true);
             if pl_attribs.menu.is_some() {
                 warn!("Setting a menu on a child window is unsupported");
             }
-            Some(parent)
-        }
-        Parent::OwnedBy(parent) => {
-            window_flags.set(WindowFlags::POPUP, true);
-            Some(parent)
-        }
-        Parent::None => {
-            window_flags.set(WindowFlags::ON_TASKBAR, true);
-            None
+            Some(handle.hwnd as HWND)
         }
+        _ => match pl_attribs.parent {
```

Comment:
`None` instead of `_`

Reply:
There might be other `RawWindowHandle` variant, so I use `_` here.
Do you want me to add more arms like `Some(_)`?